src/components/Todo/EditCard/edit-buttons.component.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import deleteButton from "assets/images/delete.png";
+import checkButton from "assets/images/check.png";
+
+const EditButtons = ({
+  handleEditTask,
+  editData,
+  taskID,
+  handleCompleteTask,
+  startDate,
+  taskData,
+}) => {
+  const handleEdit = () => {
+    handleEditTask(taskID, editData);
+  };
+  const handleComplete = () => {
+    handleCompleteTask(taskID, startDate);
+    handleEditTask(taskID, editData);
+  };
+  const handleCancel = () => {
+    handleEditTask(taskID, taskData);
+  };
+  return (
+    <div className="edit-card__btn-section">
+      <button className="todo-save__btn" onClick={handleEdit}>
+        Save
+      </button>
+      <button>
+        <img src={checkButton} alt="Complete Button" onClick={handleComplete} />
```

Comment:
Could we use const for alt value ?
```suggestion
        <img src={checkButton} alt={COMPLETE_BUTTON} onClick={handleComplete} />
```